Source/UltraleapTrackingCore/Private/OpenXRToLeapWrapper.cpp

```diff
@@ -15,7 +15,7 @@
 #include "LeapUtility.h"
```

Comment:
is this file supposed to be here ? may need to remove (looks like openxr)

Reply:
I think internally OpenXR is the default?

Reply:
So keep it.